docs/billing-integration.md

```diff
@@ -263,11 +266,43 @@ Additionally, if you've configured a `disk` endpoint, you can inspect the conten
 
 ### Getting a Reporting Secret for Testing
 
-There's currently no way to create a valid reporting secret for your application until it's published in GCP Marketplace (it doesn't need to be published publicly). You can use [this](https://storage.cloud.google.com/cloud-marketplace-tools/reporting_secrets/fake_reporting_secret.yaml) fake secret file which is structurally correct, but will fail authentication. Use the following procedure to get a valid secret for testing:
+#### Using a fake reporting secret
+
+We provide a [fake reporting secret](https://storage.cloud.google.com/cloud-marketplace-tools/reporting_secrets/fake_reporting_secret.yaml)
+which you can use for testing. This secret is structurally correct, but will fail authentication.
```

Comment:
"... which you can use for testing. This secret is structurally correct and will allow ubbagent to start collecting metrics, but will fail authentication Service Control."